projects/valtimo/process-management/src/lib/process-management-builder/process-management-builder.component.ts

```diff
@@ -164,12 +172,16 @@ export class ProcessManagementBuilderComponent implements OnInit, OnDestroy {
   }
 
   loadProcessBpmn() {
-    this.processService.getProcessDefinitionXml(this.selectedVersion.id).subscribe(xml => {
-      this.bpmnModeler.importXML(xml['bpmn20Xml']);
+    this.processService.getProcessDefinitionXml(this.selectedVersion.id).subscribe(result => {
+      this.bpmnModeler.importXML(result['bpmn20Xml']);
+      this.bpmnViewer.importXML(result['bpmn20Xml']);
+      this.isReadOnlyProcess$ = of(result.readOnly);
```

Comment:
Change to BehaviourSubject and use .next instead of reassigning the class property